### PR TITLE
Create a `command_bar` partial & render it on admin pages

### PR DESCRIPTION
**Pull request by sampoder**
## Summary of the problem

Raised by Max here: https://github.com/hackclub/hcb/pull/4432#issuecomment-1855135637

## Describe your changes

1) Made a partial to prevent repeating code
2) Rendered that partial on admin pages
3) Added a bit of CSS to ensure the colours are correct

### CODE REVIEW

File: app/views/application/_command_bar.html.erb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<% @admin_urls = current_user&.admin? ? {
```
Comment (garyhtou) [inline app/views/application/_command_bar.html.erb]: we can use `admin_signed_in?`

File: app/views/application/_command_bar.html.erb
Comment (garyhtou) [inline app/views/application/_command_bar.html.erb:1]: it looks like we lost the `if` that hides this for non-admins

File: app/views/application/_command_bar.html.erb (hunk 1 above)
Comment (sampoder) [reply app/views/application/_command_bar.html.erb]: we can!

```suggestion
<% @admin_urls = admin_signed_in? ? {
```

File: app/views/application/_command_bar.html.erb
Comment (sampoder) [reply app/views/application/_command_bar.html.erb:1]: Good catch, thanks!

**Review by garyhtou — APPROVED** [review]
great!

File: app/views/application/_command_bar.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<% if admin_signed_in? %>
+   <% @admin_urls = current_user&.admin? ? {
+        "Applications"              => "https://airtable.com/tblctmRFEeluG4do7/viwGhv19cV1ZRj61a",
+        "OnBoard ID"                => "https://airtable.com/app4Bs8Tjwvk5qcD4/tblVZwB8QMUSDAd41/viwJ15CT6VHCZ0UZ4",
+        "Wires"                     => "https://airtable.com/tbloFbH16HI7t3mfG/viwzgt8VLHOC82m8n",
```
Comment (sampoder) [inline app/views/application/_command_bar.html.erb]:
```suggestion
```

Comment (garyhtou) [reply app/views/application/_command_bar.html.erb]: good catch

**Pull request merged**